### PR TITLE
 Select job with the job id given as query parameter

### PR DESCRIPTION
**Pull request by chloegugli**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 1]
```diff
@@ -191,6 +202,22 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
         return base;
     }
 
+    private boolean selectJobIdFromUrl() {
+        String urlJobId = Window.Location.getParameter("job");
+        if (urlJobId == null) {
+            return true;
+        }
```
Comment (aminelouati) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: you can replace this if by: return (urlJobId != null)

Comment (chloegugli) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: Are you sure? The rest of the function won't be executed

Comment (aminelouati) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: yes you are right. Sorry

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 2]
```diff
@@ -129,6 +134,12 @@ public void jobsUpdated(Map<Integer, Job> jobs) {
         this.ds.setCacheData(data.toArray());
         data.destroy();
         applyCurrentLocalFilter();
+
+        //select the job given in the URL if it has not been automatically selected before
+        if (!isJobFromUrlAutoSelected && selectJobIdFromUrl()) {
+            isJobFromUrlAutoSelected = true;
+            LogModel.getInstance().logImportantMessage("isJobFromUrlAutoSelected = false");
```
Comment (fviale) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: is this log really important to keep ?

**Pull request merged**